stm/stm.ml

```diff
@@ -27,6 +27,8 @@ open Feedback
 open Vernacexpr
 open Vernacextend
 
+module PG_compat = Vernacstate.Proof_global [@@ocaml.warning "-3"]
```

Comment:
Can't you just write
```
module Vernacstace = struct
  module Proof_global = Vernacstate.Proof_global [@@ocaml.warning "-3"]
end
```
In order to avoid all the renaming below?

Reply:
That's another choice but you'd have to `include Vernacstate` in full as not to hide the API, which has a bad impact on code size; on the other hand a module alias is free.

Reply:
> which has a bad impact on code size; on the other hand a module alias is free.

How much is the binary size increased? 10Kb? 100Kb? What does "bad" mean?
It is not for free, since you have to rename things.